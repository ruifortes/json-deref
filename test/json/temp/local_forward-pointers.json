{
  "a": {
    "a": {
      "a": "val aaa",
      "b": {"$ref" : "#a/a/c"},
      "c": "val aac"
    },
    "b": {"$ref" : "#b/a"}
  },
  "b": {
    "a": "val ba",
    "b": "val bb"
  }

}
