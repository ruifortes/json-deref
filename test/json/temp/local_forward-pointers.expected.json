{
  "a": {
    "a": {
      "a": "val aaa",
      "b": "val aac",
      "c": "val aac"
    },
    "b": "val ba"
  },
  "b": {
    "a": "val ba",
    "b": "val bb"
  }
}
